great_expectations/data_context/data_context.py

```diff
@@ -2268,7 +2268,7 @@ def get_expectation_suite(
                 resource_type="expectation_suite", ge_cloud_id=ge_cloud_id
             )
         else:
-            key: ExpectationSuiteIdentifier = ExpectationSuiteIdentifier(
+            key: Union[ExpectationSuiteIdentifier, None] = ExpectationSuiteIdentifier(
```

Comment:
@Shinnnyshinshin This should be replaced by `Optional[ExpectationSuiteIdentifier]` -- thanks!

Reply:
Oh right! thank you for catching this @alexsherstinsky